docs/orchestration/index.md

```diff
@@ -56,68 +52,39 @@ applied is: `change detection`, `current directory`, `wants`.
 
 ## Stacks ordering
 
-Sometimes stacks are completely independent of each other, but on
-certain occasions it may happen that infrastructure that is created
-by **stack-a** is required by **stack-b**, like using the outputs
```

Comment:
@katcipis @mariux is this actually correct? I thought we never use outputs of one stack in another?

Reply:
we ar enot using the outputs in terramate but people can query then with remote state..